### PR TITLE
fix(learn): Updated challenge instructions to be explicit

### LINKED ISSUE #40031: Problem with Applied Visual Design: Set the font-size for Multiple Heading Elements challenge
So, I tried submitting the following code:
```
<style>






</style>
<h1 style="font-size: 68px;">This is h1 text</h1>
<h2 style="font-size: 52px;">This is h2 text</h2>
<h3 style="font-size: 40px;">This is h3 text</h3>
<h4 style="font-size: 32px;">This is h4 text</h4>
<h5 style="font-size: 21px;">This is h5 text</h5>
<h6 style="font-size: 14px;">This is h6 text</h6>
```
When I ran it through the tests, all but the last one passed. Then I added in what I was supposed to do without removing the inline styles:
```
<style>
 h1 {
    font-size: 68px;
  }
  h2 {
    font-size: 52px;
  }
  h3 {
    font-size: 40px;
  }
  h4 {
    font-size: 32px;
  }
  h5 {
    font-size: 21px;
  }
  h6 {
    font-size: 14px;
  }
</style>
<h1 style="font-size: 68px;">This is h1 text</h1>
<h2 style="font-size: 52px;">This is h2 text</h2>
<h3 style="font-size: 40px;">This is h3 text</h3>
<h4 style="font-size: 32px;">This is h4 text</h4>
<h5 style="font-size: 21px;">This is h5 text</h5>
<h6 style="font-size: 14px;">This is h6 text</h6>
```
This one passed all the tests! I know we were supposed to put stuff in the style tag, but I discovered this issue because I didn't want to do so much typing. There doesn't seem to be some check in the last test that looks at inline styles, unlike the rest. Can you look into it?

### PR DESCRIPTION
**Pull request by stevenwalker91**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #40031

Updated the instructions to be explicit around asking campers to type their code in the styles tag rather than inline styles for the Applied Visual Design: Set the font-size for Multiple Heading Elements challenge.

### CODE REVIEW

**Review by naomi-lgbt — APPROVED** [review]
LGTM~! 🥳 Congratulations on your first pull request!

As a note, our workflow requires at least one additional approval before this can be merged.

**Review by majestic-owl448 — APPROVED** [review]
LGTM! and I am going to merge this, so congratulations on your first contribution!

**Pull request merged**
